plugins/MobileMessaging/Controller.php

```diff
@@ -155,4 +158,32 @@ public function getCredentialFields()
 
         return $view->render();
     }
+
+    public function getBrowserNotifications()
+    {
+        $login = null;
+        $token = Common::getRequestVar('token');
+
+        if ($token) {
```

Comment:
Need to check how we can prevent brute force here. 

In theory we could add something like this ` 'API.MobileMessaging. getBrowserNotifications'   => 'beforeLoginCheckBruteForce',` 
here: https://github.com/matomo-org/matomo/blob/3.10.0-b3/plugins/Login/Login.php#L52

And then we would also need to call `BruteForceDetection::addFailedAttempt` on a failed attempt. The problem: If the service worker plays up and doesn't have the right token, it may cause the user to be no longer able to log in as the IP would be always blocked. However, if we request the notifications only once per hour or only a few times per day anyway then it shouldn't be a problem.

Btw: should the method be more like `getNotifications` or `getUserNotifications` since the type is no longer called `browser`?